README.md

```diff
@@ -1,33 +1,32 @@
-# Project
+# Azure Terraform Verified Modules
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+> This repo contains the source code of all currently available Terraform modules verified by Microsoft Azure. All verified modules will meet the quality pipeline established by Microsoft Azure team, which guarantee their high quality.    
 
-As the maintainer of this project, please make a few updates:
+## Modules
+Below is a table containing all Terraform modules verified by Microsoft Azure. Each module version badge shows the latest version of the corresponding module. You may click on a version badge to check all available versions for a specific module. Each minimum Terraform version shows the minimum entry requirements for the version of Terraform. Using the verified modules below the required Terraform version might cause potential inconsistency and disruption. 
+> For those with parallel modules, we strongly recommend you transfer your existing infrastructure to the new verified modules, as old parallel modules will no longer be updated. 
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+<!-- Begin Module Table -->
+
+| Module            | Module Version | Minimum Terraform Version | Docs                                                 |Key Metrics                                                 | Version Details|
+| ----------        | :-----------:  | :-----------:             |----------                                            | :-----------:                                             |:-----------:   |
+| Terraform-AzureRM | 1.2.0          | 1.13.0                    |https://github.com/Azure/terraform-azurerm-subnets    | daily downloads     weekly downloads  yearly downloads    |docs            |
 
-## Contributing
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+## Contributing
+
+We only accept contributions from Microsoft employees at this time. Teams within Microsoft can refer to Contributing to Terraform Verified Modules Registry for more information. External customers can propose new modules or report bugs by opening an issue.  
```

Comment:
I know there is not a lot of content in the contributing section, yet, but let's have a separate CONTRIBUTE.md file where we define these details. All of the content below should be in that file and this section should provide link to that doc.

Additionally, you have a copy paste issue in the second sentance: "Teams within Microsoft can refer to Contributing to Terraform Verified Modules ~~registry~~ for ..." **(remove registry)**

Also, should provide a link to where external folks can open Issues.